src/compas/files/ply.py

```diff
@@ -401,33 +414,32 @@ def read_faces_binary(self):
 
 class PLYParser(object):
     """"""
-
     def __init__(self, reader, precision=None):
         self.precision = precision
-        self.reader    = reader
-        self.vertices  = None
-        self.edges     = None
-        self.faces     = None
+        self.reader = reader
+        self.vertices = None
+        self.edges = None
+        self.faces = None
         self.parse()
 
     def parse(self):
-        self.vertices = [(vertex['x'], vertex['y'], vertex['z']) for vertex in self.reader.vertices]
-        self.faces = [face['vertex_indices'] for face in self.reader.faces]
-
+        self.vertices = [(vertex['x'], vertex['y'], vertex['z'])
+                         for vertex in self.reader.vertices]
+        self.faces = [face[self.reader.face_properties[0][0]] for face in self.reader.faces]
```

Comment:
This change is because while the definition says `vertex_indices ` Rhino uses `vertex_index ` instead.